packages/editor/tests/setupTestDependencies.ts

```diff
@@ -0,0 +1,43 @@
+/*
+CPAL-1.0 License
+
+The contents of this file are subject to the Common Public Attribution License
+Version 1.0. (the "License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+https://github.com/ir-engine/ir-engine/blob/dev/LICENSE.
+The License is based on the Mozilla Public License Version 1.1, but Sections 14
+and 15 have been added to cover use of software over a computer network and 
+provide for limited attribution for the Original Developer. In addition, 
+Exhibit A has been modified to be consistent with Exhibit B.
+
+Software distributed under the License is distributed on an "AS IS" basis,
+WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for the
+specific language governing rights and limitations under the License.
+
+The Original Code is Infinite Reality Engine.
+
+The Original Developer is the Initial Developer. The Initial Developer of the
+Original Code is the Infinite Reality Engine team.
+
+All portions of the code written by the Infinite Reality Engine team are Copyright © 2021-2023 
+Infinite Reality Engine. All Rights Reserved.
+*/
+
+import { HyperFlux, HyperStore } from '@ir-engine/hyperflux'
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+
+i18n.use(initReactI18next).init({
+  lng: 'en',
+  fallbackLng: 'en',
+  resources: {
+    en: {
+      translation: {}
+    }
+  },
+  interpolation: { escapeValue: false }
+})
+
+HyperFlux.store = <HyperStore>{
```

Comment:
why are we creating a hyperflux store here? the test setup context does not need it. also, the store is created with `createHyperStore`

Reply:
When I ran test without `HyperFlux.store` I get the following error:
```
     TypeError: Cannot read properties of undefined (reading 'stateMap')
```

That being said, I myself may be missing some knowledge on how to properly mock/stub things in unit tests like this, so any guidance would be very much appreciated! 🙏🏽

Reply:
You need to run either `createHyperStore` or `createEngine` (depending what you're testing) inside the `before` hook of the test

Reply:
Just to close this thread, I'll be removing the `toolbar.test.tsx` entirely for now, and plan to come back to this in the near future. Refer to my comment [here](https://github.com/ir-engine/ir-engine/pull/291#issuecomment-2375814339).

I will keep this in mind, however, for when I come back!